plaidml/bridge/openvino/ops/prior_box.cpp

```diff
@@ -0,0 +1,444 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "plaidml_ops.hpp"
+#include "plaidml_util.hpp"
+
+#include "ngraph/opsets/opset.hpp"
+#include "ngraph/opsets/opset4.hpp"
+
+#include "plaidml/op/op.h"
+
+using namespace plaidml;  // NOLINT[build/namespaces]
+using ngraph::opset4::PriorBox;
+
+namespace PlaidMLPlugin {
+
+class PriorBoxImpl {
+ public:
+  PriorBoxImpl(const Context& context, const ngraph::op::PriorBoxAttrs& attributes);
+
+  // Fixed size path uses fixed_size, density, fixed_ratio | aspect_ratio
+  void processFixedSizePath();
+
+  // Min size path uses min_size, max_size, aspect_ratio, scale_all_sizes
+  void processMinSizePath();
+
+  // Append variance after former result
+  void processVariance();
+
+  edsl::Tensor& getOutput() { return C_out; }
+
+ private:
+  // Preprocess data with some attributes
+  void prepareConfig();
+
+  // Produce a tensor which data are interlaced with W and H
+  // Use this to produce box center for different path
+  void produceCenterBase();
+
+  const Context& ctx;
+  const ngraph::op::PriorBoxAttrs& attrs;
+  int H;
+  int W;
+  int IH;
+  int IW;
+  int num_of_prior;
+  int min_element_size;
+  float step;
+  float step_x;
+  float step_y;
+  float offset;
```

Comment:
Generally I think we should use the highest precision floating point type (`double`) for constants and values computed at eDSL construction/compile time. Is there a reason why this wouldn't be possible in this context?

Reply:
follow https://github.com/plaidml/openvino/blob/plaidml/ngraph/core/reference/include/ngraph/runtime/reference/prior_box.hpp#L95 to get same results, can be flexible if close the comparation.

Reply:
Make step as tensor for the change of IW && IH